ui/pages/import-token/import-token.component.js

```diff
@@ -191,13 +199,24 @@ class ImportToken extends Component {
       tokenSelectorError: null,
       symbolAutoFilled: false,
       decimalAutoFilled: false,
+      mainnetTokenWarning: null,
     });
 
     const addressIsValid = isValidHexAddress(customAddress, {
       allowNonPrefixed: false,
     });
     const standardAddress = addHexPrefix(customAddress).toLowerCase();
 
+    let isMainnetToken = false;
+
+    Object.keys(contractMap).forEach((key) => {
+      if (key.toLowerCase() === customAddress.toLowerCase()) {
+        isMainnetToken = true;
+      }
+    });
```

Comment:
Nit: this code can be simplified with the `.some` method

```suggestion
    const isMainnetToken = Object.keys(contractMap).some((key) => key.toLowerCase() === customAddress.toLowerCase());
```

Reply:
updated